bump.js

```diff
@@ -3,30 +3,40 @@
 const fs = require('fs')
 const path = require('path')
 const _template = require('lodash.template')
+const semver = require('semver')
 
 const { PR_TITLE_PREFIX } = require('./const')
-const { runSpawn } = require('./util')
+const { runSpawn } = require('./utils/runSpawn')
+const { callApi } = require('./utils/callApi')
 
 const actionPath = process.env.GITHUB_ACTION_PATH
 const tpl = fs.readFileSync(path.join(actionPath, 'pr.tpl'), 'utf8')
 
 const getPRBody = (template, { newVersion, draftRelease, inputs }) => {
+  const tagsToBeUpdated = []
+  const { major, minor } = semver.parse(newVersion)
+
+  if (major !== 0) tagsToBeUpdated.push(`v${major}`)
+  if (minor !== 0) tagsToBeUpdated.push(`v${major}.${minor}`)
+
   // Should strictly contain only non-sensitive data
   const releaseMeta = {
     id: draftRelease.id,
     version: newVersion,
     npmTag: inputs['npm-tag'],
     opticUrl: inputs['optic-url'],
+    tagsToUpdate: tagsToBeUpdated.join(', '),
```

Comment:
Why are we storing these in `releaseMeta`? This is an object that's used to carry information across between the time the PR is created to when it's merged, but it doesn't look like we're using this, because when we're releasing we're using the semver again to extract the tags we need to update. Hence, we have 2 options:

- either we don't store them in releaseMeta and we simply provide them as an input to `template()`
- we store them in releaseMeta and we populate them conditionally based on whether inputs['sync-semver-tags'] is true or not, but then in the release we read them from releaseMeta instead of computing them again

Reply:
Good catch, I've gone with the first option